### PR TITLE
Fix auth on macOS High Sierra

### PR DESCRIPTION
**Pull request by satoru-people**

### CODE REVIEW

**Comment by naturallight** [comment]
+1

**Comment by naturallight** [comment]
I have auth problem with macOS High Sierra  using the corkscrew2.0.0. This problem sucks me a lot

Then I found this PR and check it out. The problem has solved.

Thank you for your PR.

**Comment by elia** [comment]
Sorry, this is not a maintained mirror of the official corkscrew library (which is located here), if you wish  (and GitHub allows it) I can transfer it to one of you, or move it to an org, otherwise I'll just archive it.

Please let me know, thanks in advance

**Comment by elia** [comment]
Ok, just checked the original domain and it's no longer available.

@patpadgett should I transfer this repo to you or to a corkscrew dedicated org?

**Comment by patpadgett** [comment]
Thank you for finding me!  It's been an amazing experience with corkscrew.  The community has maintained this small bit of code since I have abandoned it; although hosting the original site for years until an unknown expiration allowed a domain squatter to snipe agroman.net.

To answer your question, there are many forks of corkscrew available even on github.  I'd like to cherry pick and merge all of the great work that others have done since my initial scrapping together of this project.  Even if you feel like your work has been insignificant, the most significant thing has been reaching back out to me -- and if you feel comfortable transferring this repo to me is a good start, I'd attribute your work (ie. digging up bones and identifying them as a dinosaur! haha!).  Assuming the distros catchup with the relocation to github, your attribution should be good for professional and personal reasons.

TL;DR - Yes, please xfer it back.  I'm going to add your name to contributors and make sure you are known within the project.

**Comment by elia** [comment]
@patpadgett thanks for the kind words! 
I indeed just re-published your work on github just because at the time I was behind a corporate proxy that was blocking both git and the corkscrew official site! 😄

I'll try to transfer it to you directly, otherwise I'll create a dedicated org and give you admin access 👍

**Comment by patpadgett** [comment]
Resolved through autotools
